usecases/objects/batch_add.go

```diff
@@ -127,15 +127,16 @@ func (b *BatchManager) validateAndGetVector(ctx context.Context, principal *mode
 			continue
 		}
 
-		class, _, err := b.schemaManager.GetCachedClass(ctx, principal, obj.Class)
+		vclasses, err := b.schemaManager.GetCachedClass(ctx, principal, obj.Class)
 		if err != nil {
 			batchObjects[i].Err = err
 			continue
 		}
-		if class == nil {
+		if vclasses[obj.Class].Class == nil {
```

Comment:
`.Class` nil or no entry in the map?

Reply:
i agree, it's a bit confusing , but depended on that the map will return the zero value of the struct, can add it though